test/integration/patients/worklist/schedule.js

```diff
@@ -2103,51 +1942,4 @@ context('schedule page', function() {
         expect(storage.lastSelectedIndex).to.equal(null);
       });
   });
-
-  specify('clinician in no workspaces', function() {
-    cy
-      .routeCurrentClinician(fx => {
-        fx.data.relationships.workspaces.data = [];
-        return fx;
-      })
-      .routeWorkspacesBootstrap()
-      .routeDirectories(fx => {
-        fx.data = [{
-          attributes: {
-            name: 'Insurance Plans',
-            slug: 'insurance',
-            value: [
-              'BCBS PPO 100',
-              'Medicare',
-            ],
-          },
-        }];
-
-        return fx;
-      })
-      .routeActions()
-      .visit('/schedule')
-      .wait('@routeActions')
-      .itsUrl()
-      .its('search')
-      .should('not.contain', 'filter[workspace]');
-
-    cy
-      .get('.list-page__filters')
-      .find('[data-workspace-filter-region]')
-      .should('be.empty');
-
-    cy
-      .get('.list-page__filters')
-      .find('[data-all-filters-region]')
-      .click();
-
-    cy
-      .get('[data-sidebar-region]')
-      .find('[data-filter-button]')
-      .should('have.length', 1)
-      .first()
-      .get('.sidebar__label')
-      .should('contain', 'Insurance Plans');
-  });
```

Comment:
Good to remove?

Looks like this just tests that the workspace filter is not shown at the top of the list and in the filters sidebar.